gfauto/whitelist.dic

```diff
@@ -242,3 +242,5 @@ writelines
 colorgrid
 logd
 gf
+SIGSEGV
+SIGABRT
```

Comment:
Was it intentional to include these as part of the PR?  (Fine if so.)

Reply:
Yeah, I accidentally pushed another change straight to master (which should now be impossible). This fix is needed before anything will pass!